both/routes.js

```diff
@@ -53,7 +52,7 @@ if(Meteor.isClient) {
     // previous (old) page. Waiting for a little bit before setting the title
     // lets the browser history look sane.
     setTimeout(function() {
-      document.title = title;
+      document.title = titleParts.reverse().join(' - ');
```

Comment:
I actually did search the String class for an `insert()`, didn't find it and rolled my eyes at the feeble JS classes...